Resources/Prototypes/_Goobstation/Entities/Clothing/OuterClothing/specific.yml

```
@@ -23,7 +23,10 @@
   - type: HeldSpeedModifier
   - type: ToggleableClothing
     clothingPrototype: ClothingHeadHelmetHEV
-  - type: Tag
+  - type: ClothingGrantComponent # Goobstation - Make sure the Freeman doesn't get cooked by supermatter
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)